### PR TITLE
Buildroot s390 image

### PR DESCRIPTION
**Pull request by nickdesaulniers**

### CODE REVIEW

**Comment by nickdesaulniers** [comment]
I'll rebase this once https://github.com/ClangBuiltLinux/boot-utils/pull/25 lands, then it should be clearer that it's just a small change to rebuild.sh and a new image file.  I would appreciate it if @eaibmz reviewed the buildroot config file; we don't need an advanced userspace image, these are for simple boot testing, then powering down the machine.

File: buildroot/s390.config [hunk 1]
```diff
@@ -0,0 +1,11 @@
+BR2_s390x=y
+BR2_OPTIMIZE_3=y
+BR2_KERNEL_HEADERS_4_4=y
+BR2_GCC_VERSION_8_X=y
+BR2_TARGET_GENERIC_ROOT_PASSWD="root"
+BR2_TARGET_GENERIC_GETTY_PORT="ttyAMA0"
```
Comment (nickdesaulniers) [inline buildroot/s390.config:6]: is this port correct? I copied this from a different arch's config, so I suppose it's likely incorrect.

Comment (eaibmz) [reply buildroot/s390.config:6]: I would suggest using `configs/qemu_s390x_defconfig` as a template or are there reasons for not doing it ?

**Comment by nickdesaulniers** [comment]
fixed in #28

**Pull request closed without merging**